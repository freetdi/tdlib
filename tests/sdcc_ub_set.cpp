
#define GDIR boost::undirectedS
#define TDIR boost::bidirectionalS

#define BACKEND_GRAPH_TYPE \
  boost::adjacency_list<boost::vecS, boost::vecS, GDIR, cfg_node>

#define INPUT_GRAPH_TYPE \
  boost::adjacency_list<boost::setS, boost::vecS, GDIR, cfg_node>

// TODO
#define CONSTRUCTOR_WIP

#include "sdcc_.cpp"
