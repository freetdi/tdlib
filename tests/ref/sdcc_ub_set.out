==test graph==
0 <--> 1 7 8 13 
1 <--> 0 2 9 14 
2 <--> 1 3 8 10 
3 <--> 2 4 9 15 
4 <--> 3 5 10 11 
5 <--> 4 6 12 14 
6 <--> 5 7 11 13 
7 <--> 0 6 12 15 
8 <--> 0 2 12 14 
9 <--> 1 3 11 13 16 
10 <--> 2 4 12 15 
11 <--> 4 6 9 14 
12 <--> 5 7 8 10 
13 <--> 0 6 9 15 
14 <--> 1 5 8 11 
15 <--> 3 7 10 13 
16 <--> 9 
17 <--> 
18 <--> 
19 <--> 
20 <--> 
====
PP+FI+TM
0 --> 17 18 19 20 
1 --> 
2 --> 
3 --> 16 
4 --> 
5 --> 
6 --> 
7 --> 1 3 4 5 
8 --> 0 2 6 7 
9 --> 8 
10 --> 9 
11 --> 10 
12 --> 11 
13 --> 12 
14 --> 13 
15 --> 14 
16 --> 
17 --> 
18 --> 
19 --> 
20 --> 
8
0: 3 7 10 13 15
1: 1 5 8 11 14
2: 5 7 8 10 12
3: 1 3 9 11 13
4: 5 6 7 11 13
5: 3 4 5 10 11
6: 1 2 3 8 10
7: 1 3 5 7 8 10 11 13
8: 1 3 5 7 8 10 13
9: 1 3 5 7 8 13
10: 1 3 7 8 13
11: 0 1 7 8 13
12: 0 1 7 8
13: 0 1 7
14: 0 1
15: 0
16: 9 16
17: 17
18: 18
19: 19
20: 20
PP+FI
0 --> 16 18 19 20 21 
1 --> 
2 --> 
3 --> 
4 --> 
5 --> 17 
6 --> 
7 --> 
8 --> 1 3 4 7 
9 --> 
10 --> 2 5 8 9 
11 --> 6 10 
12 --> 11 
13 --> 12 
14 --> 13 
15 --> 14 
16 --> 15 
17 --> 
18 --> 
19 --> 
20 --> 
21 --> 
8
0:
1: 3 7 10 13 15
2: 5 6 7 11 13
3: 5 7 8 10 12
4: 3 4 5 10 11
5: 1 3 9 11 13
6: 0 1 7 8 13
7: 1 2 3 8 10
8: 1 3 5 7 8 10 11 13
9: 1 5 8 11 14
10: 1 3 5 7 8 11 13
11: 1 3 5 7 8 13
12: 1 3 5 7 13
13: 1 3 7 13
14: 3 7 13
15: 3 7
16: 3
17: 9 16
18: 17
19: 18
20: 19
21: 20
